tensorflow/java/src/gen/resources/tftypes.csv

```diff
@@ -1,21 +1,21 @@
 TF type,datatype code,Java type,Java boxed type,Creator?,default,Description
-Float,1,float,Float,y,0f,32-bit single precision floating point
-Double,2,double,Double,y,0,64-bit double precision floating point
+Float,1,float,Float,y,0f,32-bit single precision floating point number
+Double,2,double,Double,y,0,64-bit double precision floating point number

```

Comment:
The datatype code is no longer needed, right? If so, better to remove them - as it removes a possibly unnecessary source of inconsistency?